spyder/plugins/application/container.py

```diff
@@ -278,9 +288,13 @@ def _check_updates_ready(self):
             box.exec_()
             check_updates = box.is_checked()
         else:
+
             if update_available:
-                header = _("<b>Spyder {} is available!</b><br><br>").format(
-                    latest_release)
+                self.application_update_status.set_status_pending()
```

Comment:
@jsbautista seems like passing `latest_release` is missing in this call, so:
```suggestion
                self.application_update_status.set_status_pending(
                    latest_release=latest_release)
```
Without that an error raises:
```
Traceback (most recent call last):
  File "C:\Program Files\Spyder\pkgs\spyder\plugins\application\container.py", line 293, in _check_updates_ready
    self.application_update_status.set_status_pending()
TypeError: set_status_pending() missing 1 required positional argument: 'latest_release'
```